cartographer_grpc/map_builder_server.cc

```diff
@@ -193,4 +198,12 @@ void MapBuilderServer::UnsubscribeLocalSlamResults(
            1u);
 }
 
+void MapBuilderServer::NotifyFinishTrajectory(int trajectory_id) {
+  cartographer::common::MutexLocker locker(&local_slam_subscriptions_lock_);
+  for (auto& entry : local_slam_subscriptions_[trajectory_id]) {
+    LocalSlamSubscriptionCallback callback = entry.second;
+    callback(nullptr);
```

Comment:
Or maybe explain the convention here or in both places. I like comments.

Reply:
Done